app/invocation/invocation_model.tsx

```diff
@@ -18,6 +18,7 @@ export default class InvocationModel {
   targets: build_event_stream.BuildEvent[] = [];
   succeeded: build_event_stream.BuildEvent[] = [];
   failed: build_event_stream.BuildEvent[] = [];
+  fetchEvents: string[] = [];
```

Comment:
This name might cause readers to think that this array stores an `Event` object -- source: I was confused for a sec when I saw the `<div>{fetchEvent}</div>` render logic, lol -- maybe slightly less ambiguous if renamed to `fetchEventUrls`? (or keep as `fetchEvents` and store Event protos instead of string. But I think the only thing we can extract from a fetch event is the URL so the first approach is probably a bit cleaner)